### PR TITLE
Update microsoftauthenticatorauthenticationmethodtarget.md

### PR DESCRIPTION
**Pull request by olena-microsoft**
Edited how requireNumberMatching is represented. @mmcla @andres-canello please review.

### CODE REVIEW

**Review by mmcla — APPROVED** [review]

File: api-reference/beta/resources/microsoftauthenticatorauthenticationmethodtarget.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ A collection of users or groups enabled to use [Microsoft Authenticator authenti
 |Property|Type|Description|
 |:---|:---|:---|
 |id|String|Object ID of an Azure AD user or group.|
-|isNumberMatchingRequired (Private Preview)|Boolean|Require the user to match the number displayed on the sign-in page to approve the MFA notification.|
+|featureSettings|List|Possible values are: `requireNumberMatching` (requires number matching for MFA notifications. Value is ignored for phone sign in notifications).|
```
Comment (davidmu1) [inline api-reference/beta/resources/microsoftauthenticatorauthenticationmethodtarget.md:21]: There should be a sentence added before the possible values that describes what the property is used for.  Sign in should be hyphenated "sign-in".

File: api-reference/beta/resources/microsoftauthenticatorauthenticationmethodtarget.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ The following is a JSON representation of the resource.
   "isRegistrationRequired": "Boolean",
   "useForSignIn": "Boolean",
   "shownContext": "String",
-  "isNumberMatchingRequired": "Boolean"
+  "featureSettings": "List"
```
Comment (davidmu1) [inline api-reference/beta/resources/microsoftauthenticatorauthenticationmethodtarget.md:47]: "List" is not a valid type. It  should either be an enumeration object or string.

Comment (davidmu1) [inline api-reference/beta/resources/microsoftauthenticatorauthenticationmethodtarget.md:47]: When removing and adding properties, entries should be added to the changlog file. Information about the changelog file: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/103153/Changelog

File: api-reference/beta/resources/microsoftauthenticatorauthenticationmethodtarget.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ A collection of users or groups enabled to use [Microsoft Authenticator authenti
 |Property|Type|Description|
 |:---|:---|:---|
 |id|String|Object ID of an Azure AD user or group.|
-|isNumberMatchingRequired (Private Preview)|Boolean|Require the user to match the number displayed on the sign-in page to approve the MFA notification.|
+|featureSettings|authenticatorAppFeatureSettings|Determines what additional settings should be applied to Microsoft Authenticator. Possible values are: `requireNumberMatching` (Requires number matching for MFA notifications. Value is ignored for phone sign-in notifications).|
```
Comment (Lauragra) [inline api-reference/beta/resources/microsoftauthenticatorauthenticationmethodtarget.md:21]: The type listed here is not defined in the doc set; this is causing an API Doctor error that will block publishing this PR. Please make sure that that type is defined.

Comment (olena-microsoft) [reply api-reference/beta/resources/microsoftauthenticatorauthenticationmethodtarget.md:21]: What's a docs set? Is there somewhere I can learn about what I'm supposed to do?

Comment (olena-microsoft) [reply api-reference/beta/resources/microsoftauthenticatorauthenticationmethodtarget.md:21]: Updated enums file.

File: api-reference/beta/resources/microsoftauthenticatorauthenticationmethodtarget.md (hunk 2 above)
Comment (olena-microsoft) [reply api-reference/beta/resources/microsoftauthenticatorauthenticationmethodtarget.md]: Thanks. Updated changelog.

**Review by davidmu1 — APPROVED** [review]

File: api-reference/beta/resources/enums.md [hunk 4]
```diff
@@ -28,6 +28,18 @@ Namespace: microsoft.graph
 |reportOnlyNotApplied|
 |reportOnlyInterrupted|
 
+
+### authenticationMode values
```
Comment (millicentachieng) [inline api-reference/beta/resources/enums.md:32]: Rename `authenticationMode` to `microsoftAuthenticatorAuthenticationMode`

**Review by Lauragra — APPROVED** [review]

**Pull request merged**